roles/servicetelemetry/tasks/component_clouds.yml

```diff
@@ -49,6 +49,11 @@
     - this_cloud.events is defined
     - this_cloud.events.collectors is defined
     - this_cloud.events is iterable
+  # TODO: it should be possible to deploy the eventing SGs when ElasticSearch
+  # is not available, but currently the template for smartgateway_events
+  # expects to have information about a local ES instance on cluster.
```

Comment:
Do we currently have any way to configure the eventing SG to use an external (non-STF-provisioned) elasticsearch instance? (I don't think we do, but) If so, this seems like it could interfere.

Reply:
Okay thanks, that's what I thought